discoverer/README.md

```diff
@@ -0,0 +1,38 @@
+SeaClouds Discoverer
+====================
+
+The SeaClouds Discoverer module provides information about cloud offering used by SeaCloudsPlatform.
+
```

Comment:
@PaoloCifariello is working on the documentation to improve it (by answering to the questions you provided).